<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>История решений</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>-5</x>
     <y>1</y>
     <width>871</width>
     <height>291</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>side_a</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>side_b</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>side_c</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>angle_A</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>angle_B</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>angle_C</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>sol</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>300</y>
     <width>311</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(182, 92, 255);
font: italic 16pt &quot;Times New Roman&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Выполнить запрос</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
